### PR TITLE
Project ready for review

### PR DESCRIPTION
**Pull request by noomdalv**
User, Post and Comments models created.
Associations made.
Validations assigned.
Tests succesfully done.

### CODE REVIEW

File: app/models/post.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+end
```
Comment (Oghenebrume50) [inline app/models/post.rb:9]: Post should have a title and a body, please add them and do not forget to validate

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments, through => :posts
```
Comment (Oghenebrume50) [inline app/models/user.rb]: User has many comments,  shouldn't be through post, you already have your comment model and table, it can stand alone

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### Status: Changes Required

Good job team this project is looking good please follow the suggested changes in the comment below and make changes, then request for a review :pray: 

**Raphael Noriode**

File: app/models/post.rb (hunk 1 above)
Comment (noomdalv) [reply app/models/post.rb:9]: fixed

**Comment by noomdalv** [comment]
Validations added to Post model
User associations fixed.

**Review by mosaaleb — APPROVED** [review]
## Well done @noomdalv, @Oghenebrume50 :clap:

#### All requirements from previous review are handled :100: 
#### You can merge :+1: 

#### Good luck with the next projects

###### Muhammad Ebeid

**Pull request merged**